systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2IsolatedST.java

```diff
@@ -1129,6 +1123,31 @@ void testKMM2RollAfterSecretsCertsUpdateScramSha(ExtensionContext extensionConte
         LOGGER.info("Messages successfully mirrored");
     }
 
+    private static void modifyKafkaUserPasswordWithNewSecret(String namespace, String kafkaUserResourceName, String customSecretSource, String customPassword) {
+        Secret userDefinedSecret = new SecretBuilder()
+            .withNewMetadata()
+                .withName(customSecretSource)
+                .withNamespace(namespace)
+            .endMetadata()
+            .addToData("password", customPassword)
+            .build();
+        kubeClient().createSecret(userDefinedSecret);
```

Comment:
You should create a Secret in the usual way how we do it for each resource in the system test module (i.e., using the resource manager).